R/mod_gene_variants.R

```diff
@@ -0,0 +1,93 @@
+# Module UI
+
+#' @title   mod_gene_variant_ui and mod_gene_variant_server
+#' @description  A shiny Module.
+#'
+#' @param id shiny id
+#' @param input internal
+#' @param output internal
+#' @param session internal
+#'
+#' @rdname mod_gene_variant
+#'
+#' @keywords internal
+#' @export 
+#' @importFrom shiny NS tagList 
+mod_gene_variant_ui <- function(id){
+  ns <- NS(id)
+
+      dashboardBody(
+        tagList(
+          
+              # selectizeInput(ns("studyName"), label = "Study Name", choices = unique(kairos::exome_data$study),
+              #                selected = unique(kairos::exome_data$study), multiple = T),
+              # 
+              selectizeInput(ns("Genes"), label = "Genes", choices = unique(kairos::maf_exome@data$Hugo_Symbol),
+                             selected = "NF1", multiple = F),
+
+                       box(title = "Positional information of variants", 
+                           status = "primary", solidHeader = TRUE,
+                           width = 800,
+                           collapsible = FALSE,
+                           plotOutput(ns('lollipop_plot'))
+                          )
+                       
+              )
+          )
+}
+
+# Module Server
+
+#' @rdname mod_gene_variant
+#' @export
+#' @keywords internal
+
+mod_gene_variant_server <- function(input, output, session){
+  ns <- session$ns
+  
+  output$lollipop_plot <- shiny::renderPlot({
+    
+    maftools::lollipopPlot(
```

Comment:
this function seems to be printing to console, if this can be turned off that might speed stuff up a bit (especially when we scale up to bigger files)! 

```   HGNC    refseq.ID   protein.ID aa.length
1: HES4    NM_021170    NP_066993       221
2: HES4 NM_001142467 NP_001135939       247
Using longer transcript NM_001142467 for now.```

Reply:
will do